environments/stg/stg.tfvars

```diff
@@ -1432,6 +1432,19 @@ frontends = [
       },
     ]
   },
+  {
+    name           = "sptribs-frontend"
+    custom_domain  = "sptribs-frontend.aat.platform.hmcts.net"
+    mode           = "Detection"
+    backend_domain = ["firewall-prod-int-palo-cftaat.uksouth.cloudapp.azure.com"]
+    global_exclusions = [
+      {
+        match_variable = "QueryStringArgNames"
```

Comment:
Do you need this exclusion?

Reply:
This project is clone of fis-dss-web, that is why we used that similarly as just in case, is that any problem ?